tests/callbacks/test_device_stats_monitor.py

```diff
@@ -69,6 +77,44 @@ def log_metrics(self, metrics: Dict[str, float], step: Optional[int] = None) ->
             for f in fields:
                 assert any(f in h for h in metrics.keys())
 
+            for f in CPU_METRIC_KEYS:
+                assert not any(f in h for h in metrics.keys()), "CPU Stats should not be included"
```

Comment:
```suggestion
                assert not any(f in h for h in metrics), "CPU Stats should not be included"
```

Reply:
also, shouldn't this be `all(...)` at L78?

Reply:
maybe I am missing something here.
cpu_stats is True by default, so here they should be present?

Reply:
Good call. This test wasn't running before, so it was passing with this mistake